packages/user/src/model/invitations/utils/getInvitationLink.ts

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const getInvitationLink = (appId: number, token: string): string => {
+  // [DU 2023-JUL-07] Todo:  Get details from app table or from config
+  return `http://localhost:20074/register/token/${token}`;
```

Comment:
this is not right, I said you to hardcode it to admin app, that does not mean localhost:20074. it will not work in develop.